scripts/update_schemastore.py

```diff
@@ -0,0 +1,71 @@
+"""Update ruff.json in schemastore.
+
+This script will clone astral-sh/schemastore, update the schema and push the changes
+to a new branch tagged with the ruff git hash. You should see a URL to create the PR
+to schemastore in the CLI. The script is separated into three blocks so you can rerun
+the schema generation in the middle before committing.
+"""
+
+# %%
+import json
+from pathlib import Path
+from subprocess import check_call, check_output
+
+schemastore_fork = "https://github.com/astral-sh/schemastore"
+schemastore_upstream = "https://github.com/SchemaStore/schemastore"
+ruff_repo = "https://github.com/charliermarsh/ruff"
+
+root = Path(check_output(["git", "rev-parse", "--show-toplevel"], text=True).strip())
+schemastore = root.joinpath("schemastore")
+ruff_json = Path("src/schemas/json/ruff.json")
+
+if not schemastore.is_dir():
+    check_call(["git", "clone", schemastore_fork, schemastore])
+    check_call(
+        [
+            "git",
+            "remote",
+            "add",
+            "upstream",
+            schemastore_upstream,
+        ],
+        cwd=schemastore,
+    )
```

Comment:
I'm tempted to say we should just use a temporary directory for SchemaStore, but is that more trouble than it's worth?

Reply:
added the default to tempdir